install.py

```
@@ -68,6 +68,54 @@ def print_step(step: str):
     print(f"\n{Colors.MAGENTA}{'▶' * 3} {step}{Colors.END}")
 
 
+def detect_fedora() -> bool:
+    """Detect if running on Fedora."""
+    try:
+        with open('/etc/os-release', 'r') as f:
```

Ruff (0.11.9)

Unnecessary mode argument

Remove mode argument

(UP015)